samples/cpp/speculative_decoding_lm/CMakeLists.txt

```diff
@@ -21,14 +22,25 @@ if(NOT TARGET nlohmann_json)
     FetchContent_MakeAvailable(nlohmann_json)
 endif()
```

Comment:
I think we need to drop those dependencies are they are not used in current version of sample